Cubical/Cohomology/EilenbergMacLane/CupProduct.agda

```diff
@@ -17,6 +17,7 @@ open import Cubical.Algebra.Ring
 open import Cubical.Algebra.CommRing
 open import Cubical.Algebra.CommRing.Instances.IntMod
 open import Cubical.Algebra.Group.Instances.IntMod
+open import Cubical.Algebra.AbGroup.Instances.IntMod
```

Comment:
Is this needed? (just wondering as nothing else is changed in the file...)

Reply:
Yes, I moved the definition of Z/2 there, which is why it's needed in so many places now.